src/main/java/codeu/controller/HashtagsServlet.java

```diff
@@ -0,0 +1,54 @@
+package codeu.controller;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import codeu.model.data.Message;
+import codeu.model.store.basic.MessageStore;
+
+public class HashtagsServlet  extends HttpServlet{
```

Comment:
Whats the difference between `HashtagsServlet` and `HashtagServlet`?

Could you add a Javadoc describing each of the classes?